lib/Sabberworm/CSS/Parser.php

```diff
@@ -480,6 +483,16 @@ private function parseNumericValue($bForColor = false) {
 		return new Size(floatval($sSize), $sUnit, $bForColor, $this->iLineNo);
 	}
 
+	private function parseLineNameValue() {
+		$this->consume('[');
+		$sName = '';
+		while(!$this->comes(']')) {
+			$sName .= $this->consume(1);
```

Comment:
Reading https://developer.mozilla.org/en-US/docs/Web/CSS/custom-ident it seems we should support certain escape sequences. We could use `parseCharacter(true)` for this.

Reply:
Should okay now. Please check.